go.mod

```diff
@@ -8,7 +8,7 @@ go 1.12
 
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
-	cloud.google.com/go v0.0.0-20160913182117-3b1ae45394a2
+	cloud.google.com/go v0.34.0
```

Comment:
Do we know why this changed?

Reply:
Looks like https://github.com/golang/oauth2/blob/master/go.mod added a go.mod file requiring this version. 

I'm afraid to ask, but what dependencies does bumping this to its preferred level bring in? See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/vendor.md#adding-or-updating-a-dependency for why leaving dependencies pinned older than their minimum preferred version is problematic

Reply:
looks like this is a tiny dependency 👍 

go ahead and bump this to it's preferred level as well:

```
hack/pin-dependency.sh cloud.google.com/go v0.34.0
hack/update-vendor.sh
```

Reply:
Yeah it looks like Kubernetes' only uses cloud.google.com/go/cloud/metadata which, since 2017 (https://github.com/googleapis/google-cloud-go/commit/43376517fff0a0afae99886e3cba67018699c873#diff-4ff99412e0bc1122c84694ea4f62b8b1), has had 0 external dependencies:

```
$ ag '"cloud.google.com/go'
pkg/cloudprovider/providers/gce/gce.go
32:     "cloud.google.com/go/compute/metadata"

pkg/cloudprovider/providers/gce/gce_instances.go
27:     "cloud.google.com/go/compute/metadata"

pkg/cloudprovider/providers/gce/gce_util.go
36:     "cloud.google.com/go/compute/metadata"

vendor/cloud.google.com/go/internal/BUILD
7:    importpath = "cloud.google.com/go/internal",

vendor/cloud.google.com/go/compute/metadata/BUILD
7:    importpath = "cloud.google.com/go/compute/metadata",

vendor/cloud.google.com/go/compute/metadata/metadata.go
20:package metadata // import "cloud.google.com/go/compute/metadata"
38:     "cloud.google.com/go/internal"

vendor/github.com/grpc-ecosystem/go-grpc-middleware/Gopkg.lock
5:  name = "cloud.google.com/go"

vendor/github.com/google/cadvisor/utils/cloudinfo/gce/gce.go
24:     "cloud.google.com/go/compute/metadata"

vendor/golang.org/x/oauth2/google/google.go
24:     "cloud.google.com/go/compute/metadata"

vendor/golang.org/x/oauth2/google/default.go
16:     "cloud.google.com/go/compute/metadata"

```